### ISSUE TITLE
notes.has_tag() not working Unknown value of _df_format: None

### ISSUE THREAD

**Issue by exc4l** [klieret/AnkiPandas#42]
MWE:
```python
from ankipandas import Collection
col = Collection(user='User 1')
notes = col.cards.merge_notes()
notes.has_tag('leech')
```
results in:
```
~\Anaconda3\lib\site-packages\ankipandas\ankidf.py in _check_df_format(self)
    185             pass
    186         else:
--> 187             raise ValueError(
    188                 "Unknown value of _df_format: {}".format(self._df_format)
    189             )

ValueError: Unknown value of _df_format: None
```
pandas version 1.1.3. 
Anki version 2.1.22

Is this reproducible? If not, is there anything I can provide you with?

**Comment by klieret** [comment]
Sorry for being late! I'll check this! Thanks a lot for opening this report :)

**Comment by klieret** [comment]
I can confirm that this is happening

**Comment by klieret** [comment]
This should be fixed as soon as I publish the new version!

**Comment by klieret** [comment]
Could you try updating your AnkiPandas installation and tell me if it works now?

Thanks again for also including a MWE. This really made it easier for me to come up with a fix.

**Comment by klieret** [comment]
@all-contributors please add @exc4l for bug

**Comment by klieret** [comment]
@all-contributors please add @exc4l for bug

**Comment by klieret** [comment]
@all-contributors please add @exc4l for bug

**Comment by CalculusAce** [comment]
@klieret 

I was trying to test out the fix to the issue with C-long that I found on Windows and discovered the fix for this issue may have created another issue.

In the pervious version, I was able to merge the notes tables into the card table after expanding the fields in the note table with fields as columns. In the current build, I am now unable to merge the field columns into the cards table, which is critical for an app I am developing (to create a dictionary of . Here is the code I was using successfully in the previous version that no longer works in the current build.

```
collect = Collection()
cards = collect.cards
notes = collect.notes
notes.fields_as_columns(inplace=True)
field_cols = [col for col in notes.columns if 'nfld' in col]
cards.merge_notes(inplace=True,columns=field_cols+['nmodel','ntags','nid'])
```

I get the following error after upgrading to the latest version of ankipandas that you released this morning:

```
"ankidf.py", line 459, in merge_notes
    setattr(ret, md, getattr(self, md))

AttributeError: 'NoneType' object has no attribute 'col'
```
